src/lint.cpp

```diff
@@ -636,6 +636,9 @@ void linter::report_error_if_assignment_is_illegal(
         this->error_reporter_->report(
             error_assignment_to_const_variable_before_its_declaration{
                 *declaration, assignment, kind});
+      } else if (kind == variable_kind::_import) {
```

Comment:
Are we not rechecking the kind parameter, we can simply move the call to the previous "import" call block.